src/classes/rests/WheatherCallout.cls

```diff
@@ -0,0 +1,18 @@
+/**
+ * Created by Mariya on 29.11.2022.
+ */
+
+public with sharing class WheatherCallout {
+
+    public static String makeCallout() {
```

Comment:
MAJOR
This method says nothing by its name. 
`makeCallout` can do literally everything, so the naming must be meaningful.
E.g. `OpenWeatherController.getWeatherByCityName()` - is very clear.

Please create two methods within this class:
`getWeatherByCityName` - expects `String cityName`
`getWeatherByCityId` - expects `String cityId`